ci/playbooks/deploy_podified_openstack.yaml

```diff
@@ -0,0 +1,52 @@
+---
+# Note make download_tools and make openstack commands run
+# in https://github.com/rdo-infra/review.rdoproject.org-config/blob/master/playbooks/crc/openstack.yaml
+# We can override the openstack operator image by defining openstack_img in the zuul job
+
+- hosts: controller
+  vars:
+    install_yamls_basedir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls"
+  tasks:
+    - name: Add crc creds
+      include_role:
+        name: rhol_crc
+        tasks_from: add_crc_creds
+
+    - name: Make sure all Openstack operators are deployed
+      ansible.builtin.shell: |
+        oc get csv --no-headers=true | grep -i "succeeded"
+      register: operator_status
+      until: operator_status.rc == 0
+      retries: 20
+      delay: 30
+
+    - name: Configure local storage
+      community.general.make:
+        chdir: "{{ install_yamls_basedir }}"
+        target: crc_storage
+        params:  &install_yamls_out
+          OUT: "{{ install_yamls_basedir }}/out"
+          NNCP_INTERFACE: enp2s0
+
+    - name: Deploy OpenStack
+      community.general.make:
+        chdir: "{{ install_yamls_basedir }}"
+        target: openstack_deploy
+        params:
+          <<: *install_yamls_out
+          OPENSTACK_REPO: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/openstack-operator"
+
+    - name: Make sure Nova api service is up and running
+      ansible.builtin.shell: |
+        oc get pods --selector service=nova-api -o jsonpath={.items[*].status.phase}
+      register: nova_service
+      until: nova_service.stdout == "Running"
+      retries: 100
+      delay: 50
```

Comment:
instead I would do
`oc rsh openstackclient openstack compute service list`
to see that nova-api is actually responding to request.

Reply:
Good suggestion. I will update it in the new patchset.